crates/repl/src/repl_sessions_ui.rs

```diff
@@ -0,0 +1,257 @@
+use editor::Editor;
+use gpui::{
+    actions, prelude::*, AppContext, EventEmitter, FocusHandle, FocusableView, Subscription, View,
+};
+use ui::{prelude::*, ButtonLike, ElevationIndex, KeyBinding};
+use util::ResultExt as _;
+use workspace::item::ItemEvent;
+use workspace::WorkspaceId;
+use workspace::{item::Item, Workspace};
+
+use crate::jupyter_settings::JupyterSettings;
+use crate::repl_store::ReplStore;
+
+actions!(
+    repl,
+    [
+        Run,
+        ClearOutputs,
+        Sessions,
+        Interrupt,
+        Shutdown,
+        RefreshKernelspecs
+    ]
+);
+actions!(repl_panel, [ToggleFocus]);
```

Comment:
We can delete this now.

Reply:
Good call! https://github.com/zed-industries/zed/pull/14983